Cargo.toml

```diff
@@ -39,7 +39,6 @@ uuid = { version = "0.7.4", optional = true, features = ["v4", "serde"] }
 [features]
 default = ["parallel"]
 parallel = ["rayon", "shred/parallel", "hibitset/parallel"]
-nightly = ["shred/nightly"]
```

Comment:
A breaking change so I bumped the minor version of `specs`

We could also leave this feature in and just make it `nightly = []` if we don't want to bump to `0.16` just yet - then I can just make this PR bump to `0.15.2`

Whatever works for `specs`!